### ISSUE TITLE
content-mathml.js affecting self-closing tags

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax#1153]
The Content MathML extension converts self-closing tags, e.g.,

```html
<math>
    <mspace linebreak="newline" /> 
</math>
```

becomes

```html
<math>
    <mi>mspace</mi> 
</math>
```

**Comment by davidcarlisle** [comment]
I think it's any empty element not just the `/>` syntax.
Seems to be inherited from my initial prototype, sorry

```
			} else if (contentMMLNode.childNodes.length==0) {
				CToP.appendToken(parentNode,'mi',contentMMLNode.localName);
```

The idea was to allow things like

`<math><sin/></math>` going to `<mi>sin</mi>` without having to list all the content elements,

but you need to protect the presentation elements that could plausibly be empty, mspace, mrow, ....

**Comment by pkra** [comment]
Thanks for your help, @davidcarlisle!

**Comment by dpvc** [comment]
Closed in favor of #1323
